src/victus.js

```diff
@@ -5,9 +5,9 @@
 */
 
 !function() {
-  var canvas, ctx, w, h, color, l, t;
-  var p = Math.PI;
-  var O = Object;
+  let canvas, ctx, w, h, color, l, t;
+  const p = Math.PI;
```

Comment:
`p` and `O` should be initialized with `let` (even though they're never reassigned) to allow for additional byte saves

Reply:
Ok, I'll do it! I would just like to know how using `let` saves bytes over `const`, do you have any reference over that or is it just by using `terser` that you've noticed that?

Reply:
it's mostly because of `terser`. using `let` will make `terser` set everything up to `p` itself to an initial value of `Math.PI` (39 bytes):
`let t,e,s,o,l,h,r,n,a=Math.PI,d=Object;`

whereas using `const` costs 6 extra bytes due to that word being added to the line (45 bytes):
`let t,e,s,o,l,h,r,n;const a=Math.PI,d=Object;`

Reply:
Woah, ok thanks!